.eslintrc.json

```diff
@@ -1,8 +1,11 @@
 {
     "root": true,
     "extends": [
-        "./es5.js",
```

Comment:
Why was this file changed? This file is the configuration of eslint to use in itself.